cmake/projects/spdlog/hunter.cmake

```diff
@@ -139,7 +139,7 @@ hunter_cmake_args(
         SPDLOG_BUILD_TESTING=OFF
         SPDLOG_BUILD_TESTS=OFF
         SPDLOG_BUILD_BENCH=OFF
-        SPDLOG_FMT_EXTERNAL=NO
```

Comment:
Please note that the initial value "NO" is also legal and was an equivalent for "OFF". [docs ref](https://cmake.org/cmake/help/latest/command/if.html#condition-syntax)